### PR TITLE
Task: base-entity

### PR DESCRIPTION
**Pull request by DangHuan2k3**
## Description

- modified .prettierrc:tabwith
- added: entities
- updated: datatype for id and interval
- format: code base new .prettierrc
- created: DB on Cloud successfully

## Checklist

- [x] Code follows the project’s code style guidelines.
- [x] Verified all related functionalities.
- [x] Updated documentation (if needed).

## Screenshots

database on cloud

## Additional Information

- Refs:
[one-to-one (typeorm)](https://orkhan.gitbook.io/typeorm/docs/one-to-one-relations)
[one-to-many (typeorm)](https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations)
[many-to-many with properties (typeorm)](https://orkhan.gitbook.io/typeorm/docs/many-to-many-relations#many-to-many-relations-with-custom-properties)

### CODE REVIEW

File: src/configs/setup-swagger.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { NextFunction, Request, Response } from 'express';
 
 const api_documentation_credentials = {
   name: 'admin',
-  pass: 'admin',
+  pass: 'admin'
 };
```
Comment (NhatHuynh-Madison) [inline src/configs/setup-swagger.ts]: move this credentials to env

File: src/main.ts [hunk 2]
```diff
@@ -20,11 +20,9 @@ async function bootstrap() {
...
-            .map((error) => Object.values(error.constraints))
-            .flat(),
-        }),
-    }),
+          details: errors.map((error) => Object.values(error.constraints)).flat()
+        })
+    })
   );
   await app.listen(port);
   console.info(`🚀 Server running on: http://localhost:${port}/api-docs`);
```
Comment (NhatHuynh-Madison) [inline src/main.ts]: try logger

File: src/middlewares/logging.middleware.ts [hunk 3]
```diff
@@ -3,8 +3,8 @@ import { Request, Response, NextFunction } from 'express';
 
 @Injectable()
 export class LoggingMiddleware implements NestMiddleware {
-	use(req: Request, res: Response, next: NextFunction) {
-		console.log(`Logging Request: ${req.method} ${req.url}`);
-		next();
-	}
+  use(req: Request, res: Response, next: NextFunction) {
+    console.log(`Logging Request: ${req.method} ${req.url}`);
```
Comment (NhatHuynh-Madison) [inline src/middlewares/logging.middleware.ts:7]: try logger

File: src/main.ts (hunk 2 above)
Comment (DangHuan2k3) [reply src/main.ts]: i resolved that

**Review by huywzz — APPROVED** [review]

**Review by NhatHuynh-Madison — APPROVED** [review]

**Review by Huy-Phan-159 — APPROVED** [review]

**Pull request merged**